x-pack/test/security_solution_playwright/.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules/
+/test-results/
+/playwright-report/
+/blob-report/
+/playwright/.cache/
+/.auth/
+.env
```

Comment:
Can we move these to the global gitignore?

Reply:
@jbudz done!